account_operating_unit/README.rst

```diff
@@ -0,0 +1,96 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+===============================
+Accounting with Operating Units
+===============================
+
+This module introduces the following features:
+* Adds the Operating Unit in the account move line.
+* Defines if the operating units are self-balanced and Inter-operating unit
+clearing account at company level.
+* When users create a journal entry with lines in different operating units,
```

Comment:
Sentence is too long and barely understandable. What about:
* journal entry with lines in different operating units are checked based on the "self-balanced" set up in OU. At the time of posting the journal entry, the corresponding lines in the Inter-operating unit clearing account are automatically created, making each OU self-balanced.